src/QsCompiler/SyntaxProcessor/VerificationTools.fs

```diff
@@ -74,14 +74,6 @@ type ResolvedType with
             | _ -> false
         this.Exists condition
 
-    /// If the given type supports equality comparison, 
-    /// returns the type of an equality comparison expression as Some (which is always Bool).
-    /// Returns None otherwise.
-    member this.supportsEqualityComparison = 
```

Comment:
What was the reason for moving this into the ExpressionVerification.fs? Given that supportsArithmetic, supportsConcatenation, and supportsIteration are here, I'd rather keep it here as a property of the type, and then make the distinction whether an *expression* can contain an equality comparison depending on the execution target as part of the expression verification.

Reply:
The reason was because I wanted all of the equality checking to be in the same place (checking supported types, and checking supported capabilities). But now I agree keeping those separate makes more sense. I reverted this change.